week0002/mattfanz/spiral.php

```diff
@@ -0,0 +1,72 @@
+<?php
+// FUNCTION
+function checkValidInput($input)
+{
+    if (!is_numeric($input) || $input <= 0) {
+        fwrite(STDERR, "The input should be numberic and > 0".PHP_EOL);
+        exit;
```

Comment:
Do you mean change to exit(-1) ?
I see on http://php.net/manual/en/function.exit.php write " Exit statuses should be in the range 0 to 254"